spec/controllers/reports/subscription_reports_for_project_owners_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Reports::SubscriptionReportsForProjectOwnersController, type: :controller do
+  describe '#index' do
```

Comment:
It's nice to be explicit about the HTTP method that the action receive, for example: `"GET #index"`